Packs/Okta/Integrations/Okta_v2/Okta_v2.yml

```diff
@@ -146,6 +146,15 @@ script:
     - contextPath: Account.Factor.Status
       description: Okta account factor status.
       type: Unknown
+    - contextPath: Okta.Metadata.X-Rate-Limit-Limit
+      description: The Rate Limit limit.
+      type: Unknown
+    - contextPath: Okta.Metadata.X-Rate-Limit-Remaining
+      description: The remaining Rate Limit.
+      type: Unknown
+    - contextPath: Okta.Metadata.X-Rate-Limit-Reset
+      description: The time to resetting the Rate Limit.
+      type: Unknown
```

Comment:
these are unknown?
Docs these are int.
Would take description right out of okta docs

https://developer.okta.com/docs/reference/rl-best-practices/#check-your-rate-limits-with-okta-s-rate-limit-headers